src/utility/replace_var.c

```diff
@@ -0,0 +1,96 @@
+/*
+** EPITECH PROJECT, 2020
+** get_var.c
+** File description:
+** get_var
+*/
+
+#include <shell.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include "utility.h"
+#include "shell.h"
+#include <string.h>
+
+char * replace_var(char *cmd, env_t *env)
+{
+    char *isolated_var = NULL;
+    char *value = NULL;
+    char *new_cmd = NULL;
+    int var_len = 0;
+
+    if (cmd[0] == '\'')
+        return (cmd);
+    isolated_var = isolate_var(cmd, env);
+    if (!isolated_var)
+        return (NULL);
+    if (isolated_var == cmd)
+        return (cmd);
+    for (; isolated_var[var_len] && isolated_var[var_len] != ' '
+    && isolated_var[var_len] != '\t'; var_len++);
+    if (!var_len && cmd[0] != '\"')
+        return (cmd);
+    value = get_var_value(isolated_var, env);
+    if (!value)
```

Comment:
It won't work if the var is not the only thing in the string "not$VAR" for example will return "" and not "not" on undefined VAR.